### PR TITLE
Adds a prepend link to the conf-list

### PR DESCRIPTION
**Pull request by heiglandreas**
That way it's easier to show people how to add a conference to the list.

For that I created a page in the [wiki](https://wiki.php.net/conferences) that the added link links to...

### CODE REVIEW

**Comment by EliW** [comment]
@bjori — This looks like a great addition to me.  Any objections?

File: conferences/index.php [hunk 1]
```diff
@@ -26,7 +26,9 @@
         }
     }
 }
-$panels = "";
+$panels = '<p class="prepend"><a href="https://wiki.php.net/conferences">Want to see your conference appear here?</a></p>';
```
Comment (salathe) [inline conferences/index.php]: I don't think the "prepend" class does anything, did you mean "panel"?

Comment (heiglandreas) [inline conferences/index.php]: No, I actually meant "prepend". Currently the "panel"-class only sets the bottom-margin. My idea was to have a different class to allow a different styling of the prepend link as it's not a conference we're linking to there. But currently there's no styling added.

Comment (salathe) [inline conferences/index.php]: That seems pointless, but it's a minor thing so... do what you want.

**Comment by salathe** [comment]
Merged. Hopefully the wiki page will be useful.

**Comment by bjori** [comment]
thanks!

**Pull request merged**